Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/HolographicButton.prefab

```diff
@@ -389,8 +389,9 @@ MonoBehaviour:
   Dimensions: 1
   CanSelect: 1
   CanDeselect: 1
-  VoiceCommand: 
+  VoiceCommand: Button
```

Comment:
Does this mean that saying "Button" will press the button? That doesn't seem like something we want in our core button, since we don't share that anywhere, and the standard way to press a button with voice is to use "select"

Reply:
The See-it, Say-it label on the Holographic button says, Say "Button". So it seemed like a bug for it not to listen for the voice command so I updated this voice command field to make it match the label. This is a discussion we were just having on a project where we have a toolbar with a set of commands that you can use voice to activate. Though it is typical to say select to trigger a button you are gazing or pointing at, if the command is global, then you want to train the users on what to say to trigger the button when they are not looking at it, that case we would want consistency between the two context.

Reply:
It's been documented. https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Documentation/README_Interactable.md#voice-commands

Reply:
I can change the see-it-say it label if we want to go back to select, but we would need to make sure the Select keyword is working.